Jetsurvey/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath(libs.android.gradlePlugin)
+        classpath(libs.kotlin.gradlePlugin)
+
+        classpath(libs.versionsPlugin)
+        classpath(libs.updaterPlugin)
+        classpath(libs.ktlintPlugin)
+        classpath(libs.spotless.gradlePlugin)
+    }
+}
+
+apply(plugin ="com.github.ben-manes.versions")
+apply(plugin ="nl.littlerobots.version-catalog-update")
+
+subprojects {
+    apply(plugin ="org.jlleitschuh.gradle.ktlint")
+    apply(plugin ="com.diffplug.spotless")
+}
```

Comment:
These could be moved to a `[plugins]` block in the version catalog and then referenced through `libs` ([as done in Tivi](https://github.com/chrisbanes/tivi/blob/5c9f8bd5404f213ef5785ccbc42a711c49715b93/build.gradle#L49-L52)) to make things a bit neater, but this approach also works fine (and matches NiA)